### PR TITLE
chore: add UX GitHub template

### PR DESCRIPTION
**Pull request by nichjones1**
### What does this PR do?
This PR creates a new template for UX issues
### How to test this PR?
Confirm a new issue can be created using the UX template

- [ ] Tests are covering the bug fix or the new feature

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/UX_Template.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      
+
+  - type: textarea
+    id: Contacts
+    attributes:
+      label: Engineering Contact
+      description: Who is the primary engineer the design team can speak with about this issue?
+
+  - type: textarea
+    id: Deadlne
```
Comment (benoitf) [inline .github/ISSUE_TEMPLATE/UX_Template.yml]: typo ?

Comment (nichjones1) [reply .github/ISSUE_TEMPLATE/UX_Template.yml]: Thanks Florent, yes it was a typo.  I've updated it accordingly

File: .github/ISSUE_TEMPLATE/UX_Template.yml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  - type: dropdown
+    id: request-type
+    attributes:
+      label: Request type
+      description: "What type of request is this?"
+      options:
+        - "A logo design"
+        - "An icon"
+        - "An infographic/chart"
+        - "a template or design for printed materials"
```
Comment (benoitf) [inline .github/ISSUE_TEMPLATE/UX_Template.yml]:
```suggestion
        - "A template or design for printed materials"
```

**Review by benoitf — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/UX_Template.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+        - "An icon"
+        - "An infographic/chart"
+        - "A template or design for printed materials"
+        - "Swag design"
+        - "Graphic design not covered by the above"
+
+  - type: dropdown
+    id: user-experience
+    attributes:
+      label: User Experience Request type
```
Comment (deboer-tim) [inline .github/ISSUE_TEMPLATE/UX_Template.yml]: Seems a little repetitive ('request type' then 'user experience request type'), and I think they'll both appear but the options seem kind of orthogonal, e.g. I could do 'request type: logo design' and 'user experience request type: website mockup'? I'm wondering if these should just be a single dropdown.

File: .github/ISSUE_TEMPLATE/UX_Template.yml [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        - "User testing"
+        - "Application mockups/designs"
+        - "Website mockups/designs"
+        - "Something else UX-related"
+      
+
+  - type: textarea
+    id: Contacts
+    attributes:
+      label: Engineering Contact
```
Comment (deboer-tim) [inline .github/ISSUE_TEMPLATE/UX_Template.yml]: Although we frequently call ourselves engineers, technically I cannot be one in Canada :) , and maybe a PM might request a logo. Maybe 'Primary contact' instead? (also on next line)

File: .github/ISSUE_TEMPLATE/UX_Template.yml [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+      description: Who is the primary engineer the design team can speak with about this issue?
+
+  - type: textarea
+    id: Deadline
+    attributes:
+      label: Deadline for request
+      description: When do you need this?  If this is for an event, please let us know the date of the evnt and any lead time you need to get materials produced.
```
Comment (deboer-tim) [inline .github/ISSUE_TEMPLATE/UX_Template.yml]: Typo in 'event'.

Comment (nichjones1) [reply .github/ISSUE_TEMPLATE/UX_Template.yml]: corrected

File: .github/ISSUE_TEMPLATE/UX_Template.yml (hunk 4 above)
Comment (nichjones1) [reply .github/ISSUE_TEMPLATE/UX_Template.yml]: changed to Primary Contact

**Review by deboer-tim — APPROVED** [review]

**Review by feloy — APPROVED** [review]

**Review by Firewall — APPROVED** [review]
